autofocus/src/main/java/org/micromanager/autofocus/AutofocusTB.java

```diff
@@ -246,37 +249,95 @@ public void run(String arg) {
          // indx =1;
          snapSingleImage();
          // indx =0;
-         //core_.setShutterOpen(false);
-         //core_.setAutoShutter(true);
+         // core_.setShutterOpen(false);
+         // core_.setAutoShutter(true);
 
          IJ.log("Total Time: " + (System.currentTimeMillis() - t0));
       } catch (Exception e) {
          IJ.error(e.getMessage());
       }
    }
 
-
-
-   //take a snapshot and save pixel values in ipCurrent_
+   // take a snapshot and save pixel values in ipCurrent_
    private boolean snapSingleImage() {
       try {
          core_.snapImage();
+         Thread.sleep(50);
          Object img = core_.getImage();
-         ImagePlus implus = newWindow();
-         // this step will create a new window iff indx = 1
-         implus.getProcessor().setPixels(img);
-         ipCurrent_ = implus.getProcessor();
+
+         int width = (int) core_.getImageWidth();
+         int height = (int) core_.getImageHeight();
+
+         ImagePlus implus;
```

Comment:
It looks like you replaced the call to `newWindow()` with the code below, but now `newWindow()` (which you also updated) is never called. Could you maybe put the code below into `newWindow()` and go back to calling it?